Makefile

```diff
@@ -60,7 +70,7 @@ server:
 
 .PHONY: spec
 spec:
-	@$(BASH_TEST) "bin/rspec ${SPEC_PATH}"
+	@$(BASH_DEV) "bin/rspec ${SPEC_PATH}"
```

Comment:
Why the change? I'm guessing because this target isn't used by CI at all, in which case maybe we should just delete it.

Reply:
it isn't used in ci, so I thought it was a developer helper, so changed it to using the same config as the rest of dev helpers (since test and dev compose files are not interchangeable)